models/ArticleManager.php

```diff
@@ -0,0 +1,395 @@
+<?php
+
+class ArticleManager extends Model
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class ArticleManager has an overall complexity of 51 which is very high. The configured complexity threshold is 50.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)